docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -59,9 +62,11 @@ including the metastore :ref:`Thrift <hive-thrift-metastore>` and :ref:`Glue
 documentation </connector/hive>`.
 
 To configure access to S3 and S3-compatible storage, Azure storage, and others,
-consult the :doc:`Amazon S3 </connector/hive-s3>` section of the Hive connector
-documentation or the :doc:`Azure storage documentation </connector/hive-azure>`,
-respectively.
+consult the appropriate section of the Hive documentation.
+
+* :doc:`Amazon S3 </connector/hive-s3>`
+* :doc:`Azure storage documentation </connector/hive-azure>`
+* :ref:`GCS <hive-google-cloud-storage-configuration>`
```

Comment:
ultimately we should consider moving all the GCS stuff into a separate page .. but that can be a separate PR